### PR TITLE
refactor ticketCreate

### PR DESCRIPTION
**Pull request by jonathanlei**
## High Level Overview of Change

refactor of ticketCreate under new transaction scheme.

### Type of Change
- [X] Refactor (non-breaking change that only restructures code)

## Test Plan
test passed

### CODE REVIEW

File: src/containers/shared/components/Transaction/TicketCreate/Simple.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const Simple: TransactionSimpleComponent = (props: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { data } = props
+  const { ticketCount } = data.instructions
+
+  return (
+    <>
+      <div className="row">
+        <div className="label ticket-count">{t('ticket_count')}</div>
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/TicketCreate/Simple.tsx:13]: Use `SimpleRow` here.

File: src/containers/shared/components/Transaction/TicketCreate/TableDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { useTranslation } from 'react-i18next'
+
+export const TableDetail = (props: any) => {
+  const { t } = useTranslation()
+  const { instructions } = props
+  const { ticketCount } = instructions
+  return (
+    <div className="ticketCreate">
+      <span className="label">{t('ticket_count')}:</span>
+      <span> </span>
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/TicketCreate/TableDetail.tsx:11]: The space can be added in the first span.

File: src/containers/shared/components/Transaction/TicketCreate/test/TicketCreateSimple.test.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+describe('TicketCreate: Simple', () => {
+  it('renders', () => {
+    const wrapper = createWrapper(TicketCreate)
+    wrapper.unmount()
+  })
+  it('renders ticket count', () => {
+    const wrapper = createWrapper(TicketCreate)
+    const ticketCount = wrapper.find('.ticket-count')
+    expect(ticketCount.length).toBe(1)
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/TicketCreate/test/TicketCreateSimple.test.tsx:15]: You can use `expectSimpleRowText` to assert the value.

File: src/containers/shared/components/Transaction/TicketCreate/test/TicketCreateSimple.test.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { createSimpleWrapperFactory } from '../../test'
+import { Simple } from '../Simple'
+import TicketCreate from './mock_data/TicketCreate.json'
+
+const createWrapper = createSimpleWrapperFactory(Simple)
+
+describe('TicketCreate: Simple', () => {
+  it('renders', () => {
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/TicketCreate/test/TicketCreateSimple.test.tsx:8]: This test is redundant.

**Comment by mvadari** [comment]
super nit (but will be important if we start deciding version numbers based on PR/commit names): refactor =/= feature

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**